src/main/java/com/example/calender/controller/MeetingRoomController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.calender.controller;
+
+import com.example.calender.dto.dtoMeetingRoom;
+import com.example.calender.entity.MeetingRoom;
+import com.example.calender.service.MeetingRoomService;
+import lombok.SneakyThrows;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/meetingroom")
+public class MeetingRoomController {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Autowired
+    private MeetingRoomService meetingRoomService;
+
+    @GetMapping()
+    List<dtoMeetingRoom> getAllMeetingRooms( ){
+        return meetingRoomService.getAllMeetingRooms().stream()
+                .map(meetingRoom -> modelMapper.map(meetingRoom,dtoMeetingRoom.class))
+                .toList();
+    }
+
+
+    @SneakyThrows
+    @GetMapping("{id}")
+    ResponseEntity<dtoMeetingRoom> getMeetingRoom(@PathVariable Long id){
+        return new ResponseEntity<>(modelMapper.map(meetingRoomService.getMeetingRoomById(id),dtoMeetingRoom.class), HttpStatus.OK);
+    }
+
+    @SneakyThrows
+    @PostMapping()
+    ResponseEntity<dtoMeetingRoom> insertMeetingRoom(@RequestBody dtoMeetingRoom dtoMeetingRoom){
```

Comment:
name of a class should be camel case and should start from an upper-case character